build.sbt

```diff
@@ -0,0 +1,307 @@
+import com.typesafe.tools.mima.plugin.MimaPlugin.mimaDefaultSettings
```

Comment:
Why move from project/Build.scala to a build.sbt? is this now the more blessed way?